nose_launchable/plugin.py

```diff
@@ -194,6 +194,10 @@ def _getCapturedStderr(self):
         return ''
 
     def _addResult(self, test, status, queueing):
+        if not hasattr(test, "test"):
+            logger.warn("This test case is skipped to report because this test doesn't have test attribute. (test id: {})".format(getattr(test, "id", None)))
```

Comment:
this test type is [nose.case.Test](https://github.com/nose-devs/nose/blob/7c26ad1e6b7d308cafa328ad34736d34028c122a/nose/case.py#L19) and it overrides unittest.TestCase. Also, unittest.TestCase has a id method ref: https://docs.python.org/3/library/unittest.html#unittest.TestCase.id